powerline/renderers/lemonbar.py

```diff
@@ -36,10 +37,26 @@ def hl(self, contents, fg=None, bg=None, attrs=None):
 		return text + contents + '%{F-B--u}'
 
 	def render(self, *args, **kwargs):
+		self.segment_info['output'] = kwargs.get('matcher_info')
```

Comment:
@S0lll0s This error is one of the reasons I suggested this solution. `segment_info` supplied on the command-line should not be modified, if you think that passing `segment_info` to renderer is useful, then you need to use at least `kwargs.pop('segment_info', {}).copy()` (note: `pop` and `copy`, your solution will just as well error out).